webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/Actions.js

```diff
@@ -9,21 +9,7 @@ import { foremanUrl } from '../../../../../foreman_tools';
 
 const Actions = ({ isLoading, isGenerating, handleSubmit, invalidFields }) => (
   <>
-    <FormGroup fieldId="actions_help" className="pf-u-pt-xl">
-      {invalidFields.length === 1 && (
```

Comment:
I don't think that removing it totally is the solution, what if there's an actual error?

Reply:
The toast is just for invalid fields which is taken care of by the form group validation.

Reply:
> Linter is complaining

Thanks Leos, Updated it